src/SourceBuild/tarball/content/Directory.Build.props

```diff
@@ -211,6 +211,11 @@
     <CliInternalBuildVersion>30000001-1</CliInternalBuildVersion>
   </PropertyGroup>
 
+  <!-- Ref-only package versions -->
```

Comment:
Why not use the same version Roslyn is building with which appears to be 4.1.0?  The reason I ask is that our goal is to build Roslyn in source-build as close to how it builds in the MSFT build.  If 4.1.0 is not in SBRP, we could add it.

Reply:
Yeah, that would be best. We only have 4.0.1 in SBRP - I'll work on adding 4.1.0.

Reply:
I've updated the PR to use 4.1.0 version of the package. However, this can't be merged until https://github.com/dotnet/source-build-reference-packages/pull/452 gets merged and flows into installer repo.

I've added the 'WIP' label so this doesn't get auto-merged.

Reply:
FYI, I typically see folks use the GH `convert to draft` functionality because that disables the ability to merge.

Reply:
The SBRP change are flowing in with https://github.com/dotnet/installer/pull/14625.